.github/actions/oblt-cli-create-ccs/action.yml

```diff
@@ -0,0 +1,55 @@
+name: 'Oblt-cli create ccs'
+description: 'Run the oblt-cli wrapper to create a CCS cluster.'
+inputs:
+  remote-cluster:
```

Comment:
I decided to use this input variable name with `-` to match the oblt-cli commands. 

We could revert to be camelcase as we do with other actions, but somehow I thought it might be better to use the same input format for those parameters matching the oblt-cli flags